### PR TITLE
Update packaging to bundle the fonts and icons (.woff2 files)

### LINKED ISSUE #729: Webpack not including the font and icon assets in the .vsix file
The .vsix file is missing the font and icon assets necessary for the Data Editor.

### PR DESCRIPTION
**Pull request by shanedell**
Closes #729

### CODE REVIEW

**Review by stricklandrbls — APPROVED** [review]
+1

Ran `yarn package`, installed the `.vsix`, and ran the data editor. The icons and fonts were visible.

**Review by NolanMatt — COMMENTED** [review]
Works now

**Review by scholarsmate — APPROVED** [review]
+1, 👍

**Review by stevedlawrence — CHANGES_REQUESTED** [review]
-1

The SpaceGrotesk font is under the SIL Open Font License: https://github.com/floriankarsten/space-grotesk#license

This is allowed by ASF by is considered weak copyleft and requires appropriate lableling:

https://www.apache.org/legal/resolved.html#weak-copyleft-licenses

Additionally, this font must be listed in the root LICENSE/NOTICE files, and because it is included in the .vsix file must also be listed in the build/package/ LICENSE/NOTICE files too.

I did not look for the licenses of the other fonts, please do that.


Also, please do not wait until an rc1 to check licenses. Licensing is something that ASF takes seriously and must not be ignored or put off until later. If dependencies are updated then licenses MUST be checked. If files are added to a repo, licenses MUST be checked. This is not negotiable and seems to be a consistent problem in the daffodil-vscode repo--I think every release has had incorrect licensing and needed an rc2. Please work to avoid that by thoroughly checking PRs.

**Comment by stricklandrbls** [comment]
> @stricklandrbls Can you provide a link to where you got the fonts and icons? This will allow me to add them to the appropriate LICENSE/NOTICE files as mentioned by @stevedlawrence

I pull both of the fonts off of fonts.google.com and the icons were a bit more difficult to obtain for offline use but ultimately comes from https://fonts.google.com/icons. 

The icons seemed to only be able to be used by adding the `href` to pull from the repository. I wanted to pull the source files since we expect some customers to use the DFDL extension in an air-gapped environment. 

I couldn't find a straight forward way to do this so I had to trace the network fetch links to the server that held the source file and then download it.

**Comment by stricklandrbls** [comment]
@shanedell Yes, those are the correct repos.

**Comment by shanedell** [comment]
@stevedlawrence I believe I added the icons and fonts to the appropriate files. I know there are still a couple missing dependencies as well but we have a ticket for those and I planned to do them in a different PR. I also noticed your issue with the incorrect license headers, we will get those resolved before 1.3.1-rc1 as well.

File: NOTICE [hunk 1]
```diff
@@ -7,3 +7,10 @@ The Apache Software Foundation (http://www.apache.org/).
 Based on source code originally developed by
 - Concurrent Technologies Corporation (https://www.ctc.com/)
 - Nteligen LLC (https://www.nteligen.com/)
+
+The following NOTICE information applies to binary components distributed with this project:
+
+- src/svelte/src/resources/icons/material-icons.woff2
+  This product bundles 'material-icons' from the above files.
+  This package is available under the Apache License v2 without a NOTICE:
+        Repository at: https://github.com/google/material-design-icons
```
[the remark this answers is not in the document]
Comment (shanedell) [reply NOTICE:16]: Oh yeah, I added it to both the `build/package/NONOTICE` and `NOTICE` file by mistake.

File: LICENSE [hunk 2]
```diff
@@ -456,3 +455,198 @@ conditions of the following licenses.
     LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
     SOFTWARE.
+
+- src/svelte/src/resources/fonts/RedHatMono-Regular.woff2
+  This product bundles the above files from the 'RedHatFont' repo. (https://github.com/RedHatOfficial/RedHatFont)
+  These files are available under the SIL Open Font License, Version 1.1:
```
[the remark this answers is not in the document]
Comment (shanedell) [reply LICENSE:462]: Gotcha missed that on my last commit. The latest now has those added to the README.

**Pull request merged**